packages/KTable/KTable.vue

```diff
@@ -532,7 +533,8 @@ export default defineComponent({
           ...rowListeners,
           ...cellListeners,
           click (e) {
-            if (e.target.tagName === 'TD' && (rowListeners.click || cellListeners.click)) {
+            const isPopoverContent = e.target.className.includes('k-popover')
+            if (!ignoredElements.includes(e.target.tagName.toLowerCase()) && !isPopoverContent && (rowListeners.click || cellListeners.click)) {
```

Comment:
Again, I think we should only ignore if the `ignoredElements` do not have a `disabled` attribute (this allows the user a way to bypass if desired)

Reply:
Something like `&& !e.target.hasAttribute('disabled')`

(Untested code... just an idea)

Reply:
Added